server/locale/FR/mutations/gpl.js

```diff
@@ -86,5 +86,53 @@ export default {
             })
         ],
         ...textLogoMutations
+    ],
+    'layout:flex': [
+        [
+            'default',
+            {
+                logo: Logo.PP_PAYPAL.WHITE,
+                headline: [
+                    {
+                        tag: 'xsmall'
+                    },
+                    {
+                        tag: 'medium'
+                    }
+                ],
+                disclaimer: ['default']
+            }
+        ],
+        [
+            'ratio:1x1',
+            {
+                subHeadline: [{ tag: 'small', br: ['achats'] }],
+                styles: [
+                    `.message__sub-headline { font-size: 1.2rem; } .message__disclaimer { bottom: 12%; }`,
+                    `@media screen and (max-width: 240px) { .message__sub-headline { font-size: 0.9rem; } }`
+                ]
+            }
+        ],
+        [
+            'ratio:1x4',
+            {
+                subHeadline: [{ tag: 'small', br: ['avec', 'et', 'paiement en'] }],
+                styles: [`.message__sub-headline { font-size: 1.2rem; }`]
```

Comment:
Can you copy and paste the ```.message__sub-headline { font-size: 1.2rem; }``` rule into ratio--1x4.css? Then I think this rule can be deleted from the styles array since it's shared.